common_test.go

```diff
@@ -7,19 +7,21 @@ import (
 	"net/http/httptest"
 	"testing"
 
-	"github.com/0xsequence/authcontrol/proto"
 	"github.com/go-chi/jwtauth/v5"
 	"github.com/stretchr/testify/require"
+
+	"github.com/0xsequence/authcontrol/proto"
 )
 
-func mustJWT(t *testing.T, auth *jwtauth.JWTAuth, claims map[string]any) string {
+func mustJWT(t *testing.T, auth *jwtauth.JWTAuth, claims map[string]any) *string {
```

Comment:
There is no need for a pointer here, empty string is enough

Reply:
This is for testing purposes, i need o pointer so i am able to send just "Bearer " in testing cases.